apps/kitchen-sink/src/ensemble/screens/forms.yaml

```diff
@@ -18,6 +18,42 @@ View:
           onResponse:
             executeCode: |
               console.log('getData', response.body);
+  menu:
```

Comment:
I think this is a super good example to have but can we move it into the layouts page instead?